spec/statement.dd

```diff
@@ -488,8 +489,10 @@ $(P
         At the start of each iteration, the variables declared by
         the $(I ForeachTypeList)
         are set to be a copy of the elements of the aggregate.
-        If the variable is $(D ref), it is a reference to the
+        If the $(I ForeachTypeAttribute) is $(D ref), it is a reference to the
         contents of that aggregate.
+        If the $(I ForeachTypeAttribute) is $(D scope), the $(I ForeachType) declaration
+        will have $(D scope) semantics.
```

Comment:
Isn't that redundant ?

Reply:
I thought it necessary because of the `ref` explanation.

Reply:
I think the `ref` is necessary because the previous line mentions a copy.